react/features/recent-list/actions.js

```diff
@@ -3,9 +3,25 @@
 import {
     _STORE_CURRENT_CONFERENCE,
     _UPDATE_CONFERENCE_DURATION,
-    DELETE_RECENT_LIST_ENTRY
+    DELETE_RECENT_LIST_ENTRY,
+    DELETE_RECENT_LIST
 } from './actionTypes';
 
+/**
+ * Deletes a recent list entry based on url and date.
+ *
+ * @param {Object} entryId - An object constructed of the url and the date of
```

Comment:
It doesn't have parameters and deletes the whole list, not an entry.

Reply:
Mybad ! wrong copy/paste !